src/main/java/io/github/cottonmc/jankson/JanksonFactory.java

```diff
@@ -83,40 +92,45 @@
 		
 		//All the things you could potentially specify with just a registry ID
 		register(builder, Activity.class,                   Registry.ACTIVITY);
-		register(builder, EntityAttribute.class,            Registry.ATTRIBUTE);
-		register(builder, Biome.class,                      BuiltinRegistries.BIOME);
 		register(builder, Block.class,                      Registry.BLOCK);
 		register(builder, BlockEntityType.class,            Registry.BLOCK_ENTITY_TYPE);
-		register(builder, BlockPlacerType.class,            Registry.BLOCK_PLACER_TYPE);
+		register(builder, BlockPredicateType.class,         Registry.BLOCK_PREDICATE_TYPE);
 		register(builder, BlockStateProviderType.class,     Registry.BLOCK_STATE_PROVIDER_TYPE);
 		register(builder, Carver.class,                     Registry.CARVER);
-		register(builder, ChunkGeneratorType.class,         BuiltinRegistries.field_26375);
 		register(builder, ChunkStatus.class,                Registry.CHUNK_STATUS);
-		register(builder, ConfiguredCarver.class,           BuiltinRegistries.CONFIGURED_CARVER);
-		register(builder, ConfiguredFeature.class,          BuiltinRegistries.CONFIGURED_FEATURE);
-		register(builder, ConfiguredStructureFeature.class, BuiltinRegistries.CONFIGURED_STRUCTURE_FEATURE);
-		register(builder, ConfiguredSurfaceBuilder.class,   BuiltinRegistries.CONFIGURED_SURFACE_BUILDER);
+		register(builder, Codec.class,                      Registry.BIOME_SOURCE);
+		register(builder, Codec.class,                      Registry.CHUNK_GENERATOR);
+		register(builder, Codec.class,                      Registry.MATERIAL_CONDITION);
+		register(builder, Codec.class,                      Registry.MATERIAL_RULE);
```

Comment:
These codec ones should be removed. 1) Jankson will get confused since they're all serialisers for the same class. 2) Codec is such a generic class that it probably shouldn't have a serialiser in the first place.